### PR TITLE
feat(Forms): add `hideDivider` prop to Iterate.Toolbar

### PR DESCRIPTION
**Pull request by andlbrei**
This PR serves two purposes, based on me trying to implement a design...

1. To be able to implement the design I need to remove the divider from the toolbar. So I introduced a prop to do this, so that I can move the toolbar around.
2. I made a POC of an implementation of our design as a story in Storybook. I would like to know your thoughts on it.  
At first I tried making another version of `ViewContainer`, but I quickly figured out that when I stripped the component down, I was left with `ArrayItemArea`, which is available for consumers, but seems to be made for internal use.

I think this serves as a good example on how to make custom iterate-based components, which is nice to have IMO.

Here is the design I want to implement as well as the POC story:
image

---

image

Storybook POC
image

(I wish the height of the `Card` component was easy to configure...)

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Iterate/Toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -48,16 +56,14 @@ export default function Toolbar({
 
   return (
     <Space
-      top="medium"
+      top="large"
```
Comment (andlbrei) [inline packages/dnb-eufemia/src/extensions/forms/Iterate/Toolbar/Toolbar.tsx:59]: I moved the `x-small` top space from the toolbar `Flex.Horizontal` and "added" it to this `top` to maintain the same top space.
The reason for this is to be able to override this `top` via the `rest` props.

**Comment by tujoworker** [comment]
Thank you for the PR 🫶 I'll look next week into it.

**Comment by andlbrei** [comment]
@tujoworker will you have time this week? 🙏

**Comment by tujoworker** [comment]
@andlbrei Yes! We will provide two new properties to make that easy for your to get.

**Comment by andlbrei** [comment]
> @andlbrei Yes! We will provide two new properties to make that easy for your to get.

Thanks!

**Comment by tujoworker** [comment]
I think we are better off with two new properties and some custom code to achieve this. Here is a PR # 4329.

**Comment by andlbrei** [comment]
@tujoworker looks good!

**Comment by andlbrei** [comment]
Solved in https://github.com/dnbexperience/eufemia/pull/4329

**Pull request closed without merging**